devstack/plugin.sh

```diff
@@ -77,10 +79,11 @@ if is_service_enabled group-policy; then
         echo_summary "Initializing $GBP"
         if [[ $ENABLE_NFP = True ]]; then
             echo_summary "Initializing $NFP"
-            [[ $DISABLE_BUILD_IMAGE = False ]] && create_nfp_image
```

Comment:
Don't remove this. Keep it.
But when the mode of installation is advanced mode, set DISABLE_BUILD_IMAGE to True